handle-multipart-upload.go

```diff
@@ -37,7 +37,7 @@ func HandleMultipartRequest(f *proxy.Flow) error {
 	// Extract the boundary from the Content-Type header.
 	contentType := f.Request.Header.Get("Content-Type")
 	boundary := strings.Split(contentType, "boundary=")[1]
-	boundary = strings.Trim(boundary, "'")
+	boundary = strings.Trim(boundary, "'\"")
```

Comment:
Have you tested with gcloud storage command? Different tools may set the boundary slightly differently, especially the delimiter.  It seems the SDK has the "'\"" at the end of boundary and it breaks the go mime/multipart library?  Can you double check, I'd imagine the multipart lib process multipart based on the spec and is quite robust.

Reply:
yes, I did test with both gcloud and SDK, gcloud was showing ' at the end of the boundary and sdk client sdk was showing " at the end of boundary, this specific change is reverted now, as it was already merged to main by bron from a different PR.